resource_booking/__manifest__.py

```diff
@@ -1,24 +1,23 @@
 # Copyright 2021 Tecnativa - Jairo Llopis
+# Copyright 2022 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {
     "name": "Resource booking",
     "summary": "Manage appointments and resource booking",
-    "version": "13.0.2.6.5",
-    "development_status": "Beta",
+    "version": "14.0.1.0.0",
+    "development_status": "Production/Stable",
     "category": "Appointments",
     "website": "https://github.com/OCA/calendar",
     "author": "Tecnativa, Odoo Community Association (OCA)",
-    "maintainers": ["Yajo"],
+    "maintainers": ["pedrobaeza"],
```

Comment:
Thanks for taking maintainership of this module. ❤️ In the future I'll appreciate it if you ping me about these kind of changes instead of just doing them without notice.

Reply:
Well, you are not anymore with us, so it's logic that you are not going to maintain this module. But if you prefer to keep maintaining it, I'm glad to leave it to you. But you can't retain something that you are not going to maintain, and you haven't expressed such desire nor do anything in this module since your departure, so I don't feel in the obligation of warning your for each module you have done under Tecnativa.